### PR TITLE
Do not show binding parameters on the overview page

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
See #2323

### CODE REVIEW

File: app/scripts/directives/serviceInstanceBindings.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ angular.module('openshiftConsole').component('serviceInstanceBindings', {
   ],
   controllerAs: '$ctrl',
   bindings: {
+    isOverview: '<?',
```
Comment (spadgett) [inline app/scripts/directives/serviceInstanceBindings.js]: Maybe call this `showParameters` and flip the logic?

**Comment by spadgett** [comment]
@jeff-phillips-18 verified the fix works

**Comment by spadgett** [comment]
But we should still look at the margins:

openshift web console 2017-10-20 11-10-44

File: app/scripts/directives/serviceInstanceBindings.js (hunk 1 above)
Comment (jeff-phillips-18) [reply app/scripts/directives/serviceInstanceBindings.js]: We can but isOverview is already used in the sub-component  (serviceBinding) so this is really just a pass thru to that.

Comment (spadgett) [reply app/scripts/directives/serviceInstanceBindings.js]: I'm OK with it then

**Comment by spadgett** [comment]
Going to leave #2323 open to track the other display issue

**Comment by spadgett** [comment]
@jeff-phillips-18 dist mismatch

**Comment by jeff-phillips-18** [comment]
Rebased to fix dist issues.

**Comment by spadgett** [comment]
still some dist problems

**Comment by jeff-phillips-18** [comment]
Added visual changes for binding parameters as well. See discussion in https://github.com/openshift/origin-web-catalog/pull/518

image

**Pull request merged**